### DISCUSSION TITLE
Installing this on coolify?

### DISCUSSION THREAD

**Discussion by boringcommerce** [mfts/papermark#394] [General]
Hello,

Is there a straightforward way to install this on coolify?

**Comment by mfts** [comment]
I never heard of or used coolify before, so I don't know.

We will update the self-hosting docs soon and make it easier to self-host. Currently it's possible with Vercel

**Reply by JSanchezFDZ** [reply]
Please push this!

**Reply by racso-dev** [reply]
@mfts Did you have the chance to do it? Generally speaking, apart from the Coolify one click deployment feature request, It would be really nice to have some docs on how to selft-host the project!

**Comment by racso-dev** [comment]
Having a straightforward way to deploy on Coolify would be really appreciated!

**Comment by jrobinfo** [comment]
Im trying to deploy on coolify now.  Im getting a 404 on /dashboard. i'll keep at it.

**Reply by dashr** [reply]
Did you ever get this going? Any initial PR of the attempt? Looking to do the same. Happy to continue the progress.
